tests/integration/fixtures/mini_sentry.py

```diff
@@ -225,6 +225,13 @@ def get_metrics(self, timeout=None):
             ),
         )
 
+    def get_outcomes(self, n, *, timeout=None):
```

Comment:
What does the `*` do?

Reply:
Best you look this up in the Python docs, I would just give an incomplete summary.